tests/bluetooth/audio/ascs/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+
+project(bluetooth_ascs)
+find_package(Zephyr COMPONENTS unittest HINTS $ENV{ZEPHYR_BASE})
+
+include_directories(
+  src
+  ${ZEPHYR_BASE}/include/zephyr
+  ${ZEPHYR_BASE}/include/zephyr/bluetooth
+  ${ZEPHYR_BASE}/include/zephyr/bluetooth/audio
+  ${ZEPHYR_BASE}/subsys/bluetooth
+  ${ZEPHYR_BASE}/subsys/bluetooth/audio
+)
+
+add_subdirectory(${ZEPHYR_BASE}/tests/bluetooth/host host_mocks)
+add_subdirectory(${ZEPHYR_BASE}/tests/bluetooth/audio/ascs/mocks mocks)
+
+target_link_libraries(testbinary PRIVATE mocks host_mocks)
+
+target_sources(testbinary
+  PRIVATE
+    src/main.c
+
+    ${ZEPHYR_BASE}/subsys/bluetooth/audio/ascs.c
```

Comment:
These files could be part of a library, as was done in controller/uut

Reply:
done